scripts/build_release_sign.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+. ${HOME}/build_env
+set -e
+set -x
+
+cd ${GOPATH}/src/github.com/algorand/go-algorand
+
+# Tag Source
+TAG=${BRANCH}-${FULLVERSION}
+if [ ! -z "${SIGNING_KEY_ADDR}" ]; then
+    git tag -s -u "${SIGNING_KEY_ADDR}" ${TAG} -m "Genesis Timestamp: $(cat ./genesistimestamp.dat)"
```

Comment:
We need to stop using the genesistimestamp for this - the timestamp no longer has real meaning (it's not the actual genesis timestamp - those are essentially hard-coded now for release builds).  Maybe the branch or channel and version.

Reply:
ok. should there be any comment at all? output of `date -u` ?

Reply:
That's fine.  Maybe more useful would be capturing some of the environment variables like CHANNEL, DEFAULT_NETWORK, DEFAULT_RELEASE_NETWORK